packages/google_sign_in/google_sign_in_android/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -720,32 +744,35 @@ private void finishWithUserData(Messages.UserData data) {
     }
 
     private void finishWithError(String errorCode, String errorMessage) {
-      Messages.Result<?> result;
```

Comment:
This had to be restructured because `voidResult` is no longer assignable to `Result<T>`.